eth/filters/filter_system.go

```diff
@@ -50,16 +50,20 @@ import (
 
 // Config represents the configuration of the filter system.
 type Config struct {
-	LogCacheSize int           // maximum number of cached blocks (default: 32)
-	Timeout      time.Duration // how long filters stay active (default: 5min)
+	IndexedLogCacheSize   int           // maximum number of indexed cached blocks (default: 32)
+	UnindexedLogCacheSize int           // maximum number of unindexed cached blocks (default: 32)
+	Timeout               time.Duration // how long filters stay active (default: 5min)
 }
 
 func (cfg Config) withDefaults() Config {
 	if cfg.Timeout == 0 {
 		cfg.Timeout = 5 * time.Minute
 	}
-	if cfg.LogCacheSize == 0 {
-		cfg.LogCacheSize = 32
+	if cfg.IndexedLogCacheSize == 0 {
```

Comment:
We should never pass 0 here based on the way configs are loaded, but we defend just in case.